README.md

```diff
@@ -502,6 +503,38 @@ Supported error codes:
 * `pb_utils.TritonError.UNAVAILABLE`
 * `pb_utils.TritonError.UNSUPPORTED`
 * `pb_utils.TritonError.ALREADY_EXISTS`
+* `pb_utils.TritonError.CANCELLED` (since 23.10)
+
+#### Request Cancellation
+
+One or more requests may be cancelled during execution, for example, cancelled
+by the user. Starting from 23.10, `request.is_cancelled()` returns up-to-date
+`True` or `False` on whether the request is cancelled. If a request is
+cancelled, the model should respond `pb_utils.TritonError.CANCELLED` in place of
```

Comment:
> If a request is cancelled, the model should respond `pb_utils.TritonError.CANCELLED`....

Is this a hard requirement? If checking for the cancelled state is optional, howcome responding in a certain way a requirement? Additionally, can't this be handled internally? As in if the request is cancelled then why the model developer should be required to create a cancelled response and return it?

Reply:
The interface is simplified. On non-decoupled model, the model may return any object on the responses list if the request is cancelled. On decoupled model, the `TRITONSERVER_RESPONSE_COMPLETE_FINAL` flag is sent automatically upon checking for cancellation on a cancelled request, so the model does not need to do anything after checking for cancellation if cancelled.